pkg/cmd/pulumi/import.go

```diff
@@ -540,8 +543,18 @@ func newImportCmd() *cobra.Command {
 				}
 				importFile = f
 			} else {
-				if len(args) < 3 {
-					return result.Errorf("an inline resource must be specified if no converter or import file is used")
+				msg := "an inline resource must be specified if no converter or import file is used, missing "
+				if len(args) == 0 {
+					contract.IgnoreError(cmd.Help())
+					return result.Errorf(msg + "type, name, and id")
+				}
+				if len(args) == 1 {
+					contract.IgnoreError(cmd.Help())
+					return result.Errorf(msg + "name and id")
+				}
+				if len(args) == 2 {
+					contract.IgnoreError(cmd.Help())
+					return result.Errorf(msg + "id")
```

Comment:
Ack, although for the manual variant, consider a switch statement?
```suggestion
				switch len(args) {
				case 0:
					contract.IgnoreError(cmd.Help())
					return result.Errorf(msg + "type, name, and id")
				case 1:
					contract.IgnoreError(cmd.Help())
					return result.Errorf(msg + "name and id")
				case 2:
					contract.IgnoreError(cmd.Help())
					return result.Errorf(msg + "id")
```

Reply:
Sure